cluster/gce/gci/configure-helper.sh

```diff
@@ -2230,8 +2230,11 @@ function start-image-puller {
     /etc/kubernetes/manifests/
 }
 
-# Starts a l7 loadbalancing controller for ingress.
+# Setups manifests for ingress controller and gce-specific policies for service controller.
 function start-lb-controller {
+  setup-addon-manifests "addons" "loadbalancing"
```

Comment:
minor: having both `cluster-loadbalancing/` and `loadbalancing/` seems confusing. Should we make a `cluster-loadbalancing/rbac` folder or something similar?

Reply:
The readme says to avoid naming conflicts. Are subdirs merged correctly?

Reply:
Sorry, didn't notice this is under cluster/gce. Now it seems good :)

Reply:
I'm open to better naming. Thankfully, it can easily be changed later when/if we add more manifests.